app/views/posts/show.html.erb

```diff
@@ -2,4 +2,8 @@
   <%= @post.tweet %>
   <hr>
   <%= @post.planned_time %>
+
+  <p>
+    <%= link_to "Tweet!", team_tweet_post_path(@post.team, @post), method: :post, class: "button" %>
```

Comment:
This is an action right? and an action can become a button, and when you use button_to it already has the method post